curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/62bb4009e3458a128ff57d5d.md

```diff
@@ -6,15 +6,19 @@ dashedName: step-68
 ---
 
 # --description--
```

Comment:
I don't know how you bypass enlist errors, but this should have given you an error that you are missing an extra space 

```suggestion
# --description--

```